cfme/tests/configure/test_ntp_server.py

```diff
@@ -0,0 +1,89 @@
+import fauxfactory
+from cfme import login
+from cfme.configure import configuration
+from datetime import datetime
+from fixtures.pytest_store import store
+from utils.conf import cfme_data
+from utils.log import logger
+from utils.ssh import SSHClient
+from utils.wait import wait_for
+
+
+def ssh_command_execution(command):
+    with SSHClient() as ssh:
+        status, msg = ssh.run_command("%s" % command)
+        return status, msg
+
+
+def appliance_date():
```

Comment:
This function, with the above change, would then become

```python
def appliance_date():
    status, msg = store.current_appliance.ssh_client.run_command("date --iso-8601=hours")
    return datetime.strptime(msg.rsplit('-', 1)[0], '%Y-%m-%dT%H')
```

Though it's longwinded, we can't really cache/shorten the store, just in case the appliance changes.